Dockerfile

```diff
@@ -100,6 +100,14 @@ ENV IMAGE_NAME=saucelabs/stt-playwright-node
 ARG BUILD_TAG
 ENV IMAGE_TAG=${BUILD_TAG}
 
+# Let saucectl know where to mount files
+RUN mkdir -p /home/seluser/__project__/ && chown seluser:seluser /home/seluser/__project__/
+LABEL com.saucelabs.project-dir=/home/seluser/__project__/
+ENV SAUCE_PROJECT_DIR=/home/seluser/__project__/
+
+# Let saucectl know what command to execute
+LABEL com.saucelabs.executable=/home/seluser/bin/folio
```

Comment:
@FriggaHel is this what saucectl will execute instead of `npm test`? what do you think about calling it `entrypoint`?

Reply:
It will later, make I'll rename it.